linux/gnome/src/main/java/com/github/weisj/darkmode/platform/linux/gnome/GnomeSettings.kt

```diff
@@ -0,0 +1,97 @@
+package com.github.weisj.darkmode.platform.linux.gnome
+
+import com.github.weisj.darkmode.platform.LibraryUtil
+import com.github.weisj.darkmode.platform.settings.*
+import com.google.auto.service.AutoService
+
+/**
```

Comment:
This notice isn't needed anymore because we aren't actually circumventing any limitations of auto-service with the provider aproach.

Reply:
Completed. See 7c6fc0c97c971afc4cb6e24cb04ce54fd918011c.